### PR TITLE
S1t5 email display container

### PR DESCRIPTION
**Pull request by mfk07**
Creation of email display and css

### CODE REVIEW

File: src/Components/EmailDisplay/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import './styles.scss';
+
+const EmailDisplay = () => {
+    return (
+        <div className="container">
+    <div className="row">
```
Comment (mikeycodingstuff) [inline src/Components/EmailDisplay/index.js]: indent wrong here but may just be github

File: src/Components/EmailDisplay/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            Hello world, <br></br>
+            This is my first email. <br></br>
+            I am looking forward to speaking with you. <br></br>
+            Could you please click this link and schedule a time and date for our call? <br></br>
+            If you have any questions about the call, just reply to this email. I will get back to you ASAP. <br></br>
+            Thank you, <br></br>
+            Maria
+            </p>
+        </div>
+
```
Comment (mikeycodingstuff) [inline src/Components/EmailDisplay/index.js]: blank line

File: src/Components/EmailDisplay/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    </div>
+
```
Comment (mikeycodingstuff) [inline src/Components/EmailDisplay/index.js]: blank line

File: src/Components/PageTemplate/index.js [hunk 4]
```diff
@@ -27,8 +28,12 @@ const PageTemplate = () => {
             <div className="col-2">
                 <SideNav />
             </div>
-            <div className="col-4">test</div>
-            <div className="col-6">test</div>
+            <div className="col-4">
+
```
Comment (mikeycodingstuff) [inline src/Components/PageTemplate/index.js]: blank line

**Review by mikeycodingstuff — COMMENTED** [review]
Looks good. Need to review font sizes - looks very small on our screens.

File: src/Components/EmailDisplay/index.js (hunk 1 above)
Comment (mfk07) [reply src/Components/EmailDisplay/index.js]: indent addressed

File: src/Components/EmailDisplay/index.js (hunk 2 above)
Comment (mfk07) [reply src/Components/EmailDisplay/index.js]: line deleted

File: src/Components/EmailDisplay/index.js (hunk 3 above)
Comment (mfk07) [reply src/Components/EmailDisplay/index.js]: line deleted

File: src/Components/PageTemplate/index.js (hunk 4 above)
Comment (mfk07) [reply src/Components/PageTemplate/index.js]: line deleted

**Comment by mfk07** [comment]
Above changes have been made as well as text sizing and spacing

**Review by mikeycodingstuff — APPROVED** [review]

**Pull request merged**